pkg/manager/diff.go

```diff
@@ -512,14 +512,54 @@ func (rr *reproRunner) Run(r *repro.Result) {
 	rr.done <- ret
 }
 
-func PatchFocusAreas(cfg *mgrconfig.Config, gitPatch []byte) {
-	const maxAffectedByHeader = 50
+func PatchFocusAreas(cfg *mgrconfig.Config, gitPatches [][]byte) {
+	direct, transitive := affectedFiles(cfg, gitPatches)
+	if len(direct) > 0 {
+		sort.Strings(direct)
+		log.Logf(0, "adding directly modified files to focus_order: %q", direct)
+		cfg.Experimental.FocusAreas = append(cfg.Experimental.FocusAreas,
+			mgrconfig.FocusArea{
+				Name: "modified",
+				Filter: mgrconfig.CovFilterCfg{
+					Files: direct,
+				},
+				Weight: 3.0,
```

Comment:
3.0 is hard to understand

Reply:
The alternative may be to use "Priority" - high, mid, low.
Feel free to ignore.

Reply:
I'd leave it as is for now. We'll likely change the approach in some time anyway and these consts are not supported by any experiments, just a guess.